content/ja/handbook/engineering/architecture/design-documents/cells/impacted_features/personal-namespaces.md

```
@@ -0,0 +1,145 @@
+メリット:
+
+- ユーザーはパーソナルプロジェクトをグループに整理できるようになります。これは非常に要望の多い機能です。
+- ユーザー作成時にパーソナル Namespace を作成する必要がなくなります。
+- 既存のパーソナルプロジェクトのパス変更が不要です。
+
+デメリット:
+
+- パーソナルグループの概念を確立する必要があります。
+- @-メンションがどのように機能するかが不明です。現在、個々のユーザーとグループの両方にタグ付けが可能です。既存のロジックに従うと、パーソナルグループに属するすべてのグループメンバーがタグ付けされます。
+- ユーザープロフィールページへの重大な影響があります。パーソナルプロジェクトはユーザープロフィールページから切り離され、ユーザーが選択した特定のプロジェクト（スターやピン留め）をハイライトする新しい機能に置き換えられる可能性があります。
+- トップレベルのグループを移行するのと同じメカニズムを使用して Organization 間でグループを移行できるかどうかが不明です。この機能は少なくとも中期的には非常に制限されると予想されます。[セクション 3.1.](#31-one-personal-namespace-that-can-move-between-organizations) で説明した転送制限と同様の制限が予想されます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)